78 testcase floating point numbers tolerance

atomczak opened the pull request.

atomczak committed:
update floating point comparison tests

removed attribute and property tests using fixed 1e6 tolerance, replaced with relative precision (according to #294)

atomczak committed:
remove old floating point comparison tests

remove omitted tests with a fixed tolerance

atomczak committed:
fix naming convention

As per #309 replace underscore with dash.

atomczak committed:
remove redundant tolerance tests

#310

atomczak committed:
Revert "remove redundant tolerance tests"

This reverts commit 3704984c4a4e3671b2104eae369204b4a8addd28.

atomczak committed:
Revert "fix naming convention"

This reverts commit da38da8e5f3abf711dabdafa0ea2523ddc58f4b7.

atomczak committed:
Merge remote-tracking branch 'upstream/development' into 78-testcase-floating-point-numbers-tolerance

atomczak committed:
fix naming convention

As per #309 replace underscore with dash.

atomczak committed:
remove redundant tolerance tests

As per #310

atomczak committed:
fix naming convention

#309

atomczak committed:
remove extraneous characters

#315

atomczak committed:
exclusive tolerance fix

as per #315, the exclusive comparison was hitting the tolerance 1e-6

atomczak committed:
fix tolerance test considering double precision

as per #315

atomczak committed:
update ids tests ifc versions

corresponds to the already updated scripts.md file

CBenghi commented:
Hello @atomczak,
I've made changes to the tooling in the latest `development` to reflect the publication of 1.0.
Could you merge `development` in this branch and incorporate the changes?

CBenghi committed:
Merge branch 'development' into proposed PR

CBenghi commented:
@atomczak,
never mind, it was simple enough to do so I did it.

CBenghi commented:
I would merge this to develop, because if fixes the naming convention, which seems critical.

At that point we will also get feedback from implementers on the nature of the tests.

CBenghi commented:
@atomczak I'll leave it to you to merge, since you are more familiar with the matter.

For this one, I think sooner rather than later. Even if it has mistakes, it is an improvement on the current state of `development`.

atomczak committed:
add tolerance documentation

based on the summary by @giuseppeverduciALMA (https://github.com/buildingSMART/IDS/blob/0d50fd8f2dbd5b388f6fafb67da255cc3ce2b4ca/Documentation/tolerance.md). #315

atomczak committed:
Merge branch '78-testcase-floating-point-numbers-tolerance' of https://github.com/buildingSMART/IDS into 78-testcase-floating-point-numbers-tolerance
